options.html

```diff
@@ -3,67 +3,90 @@
 	<head>
 		<title>Video Speed Controller: Options</title>
 		<link rel="stylesheet" href="options.css" />
-		<script src="options.js"></script>
+        <script src="jquery-3.2.1.slim.min.js"></script>
```

Comment:
Could we avoid it? :-)

Reply:
for making/testing faster I added jquery. I'm not sure about if I can edit all usages easily. Do we need to remove jquery since it is just for settings page?

Reply:
Which are the parts that would be more challenging to replace?